features/claims/litigator/litigator_trial_claim_draft_submit.feature

```diff
@@ -13,11 +13,13 @@ Feature: Litigator partially fills out a draft final fee claim, then later edits
 
     And I should see 3 supplier number radios
 
-    When I choose the supplier number '1A222Z'
     And I select a case type of 'Trial'
     And I select the court 'Blackfriars'
     And I enter a case number of 'A20161234'
     And I enter the case concluded date '2016-04-01'
+    Then I click "Continue" in the claim form and move to the 'Case details' form page
```

Comment:
is this expecting an error? if so you could test for that. also should be `move to the 'Defendants details' form page`

Reply:
Yep, we're expecting an error and we will add in a test for it.

Reply:
@jsugarman see subsequent commit